test/blackbox-tests/test-cases/copy_files.t/run.t

```diff
@@ -44,3 +44,19 @@ Test (alias ...) and (mode ...) fields:
 
   $ cat test3/foo.txt
   Foo
+
+Test external paths:
+
+  $ mkdir -p test4
+  $ cat >test4/dune-project <<EOF
+  > (lang dune 2.7)
+  > EOF
+  $ P=1234567890.txt
```

Comment:
What about using `mktemp` to create the temp file?

This will respect `TMPDIR` (or `TEMP` on windows) and will also not break when we run than test suite per a version of OCaml at once.

Reply:
Thanks, fixed.